spec/decorators/odk/form_decorator_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+# Form decorator spec
```

Comment:
This is not necessary since it's obvious from the next line. Only comment things that aren't obvious. Prefer descriptive code where possible, and that's what you have below. :)

Reply:
alrighty
Was trying to avoid rubocop lint warning